reverb/cc/platform/default/build_rules.bzl

```diff
@@ -309,10 +309,16 @@ def reverb_gen_op_wrapper_py(name, out, kernel_lib, linkopts = [], **kwargs):
             "-fvisibility=hidden",  # avoid symbol clashes between DSOs.
         ],
         linkshared = 1,
-        linkopts = linkopts + _rpath_linkopts(module_name) + [
-            "-Wl,--version-script",
-            "$(location %s)" % version_script_file,
-        ],
+        linkopts = linkopts + _rpath_linkopts(module_name) + select({
+            "//reverb:macos": [
```

Comment:
i'm not sure this is enough.

on macs, you should probably be using "-exported-symbols-list" and generating an exported_symbols.lds in addition or instead of version_script.lds

https://github.com/tensorflow/tensorflow/blob/7e52814467d499776f6cfb9703bded0117bc5424/tensorflow/BUILD#L777

the format of the exported_symbols.lds differs slightly from that of version_script so you'll have to generate it differently.

for example:

https://github.com/tensorflow/tensorflow/blob/2aefdee7b7d691c2be1e08832142c1c28add312b/tensorflow/lite/c/exported_symbols.lds

vs

https://github.com/tensorflow/tensorflow/blob/2aefdee7b7d691c2be1e08832142c1c28add312b/tensorflow/lite/c/version_script.lds